development/build/scripts.js

```diff
@@ -1223,7 +1223,7 @@ async function setEnvironmentVariables({
     DEBUG: devMode || testing ? variables.getMaybe('DEBUG') : undefined,
     EIP_4337_ENTRYPOINT:
       variables.getMaybe('EIP_4337_ENTRYPOINT') ||
-      '0x5FF137D4b0FDCD49DcA30c7CF57E578a026d2789',
+      (testing ? '0x18b06605539dc02ecD3f7AB314e38eB7c1dA5c9b' : undefined),
```

Comment:
Should we import this from test/e2e/constants.ts?

Reply:
Unfortunately we can't since this isn't part of the extension bundle, but the build system.